automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -286,7 +286,9 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
   def createNewCluster(googleProject: GoogleProject, name: ClusterName = randomClusterName, request: ClusterRequest = defaultClusterRequest, monitor: Boolean = true)(implicit token: AuthToken): Cluster = {
     val cluster = createCluster(googleProject, name, request, monitor)
     if (monitor) {
-      cluster.status shouldBe (if (request.stopAfterCreation.getOrElse(false)) ClusterStatus.Stopped else ClusterStatus.Running)
+      withClue(s"Monitoring Cluster status: $name") {
```

Comment:
when failed, this is the error that would be logged to file.
`org.scalatest.exceptions.TestFailedException: Monitoring Cluster status: ClusterName(automation-test-aarrttq3z) Error was not equal to Running
`